### ISSUE TITLE
Handoff to Ashton for finalizing sound design

### ISSUE THREAD

**Issue by emily-phet** [phetsims/ohms-law#103]
Would like Ashton to start working on a final sound design for this sim - based on Mike and JB's existing work. 

Note: we need to do user testing with users of screen readers to get feedback on "current" sound, and how overlap of text/sound is working (or not) to support their experience - before completely finalizing sound (started another issue for this). Would probably be best if we can go ahead and do user testing with more finalized version if possible, and make tweaks from there.

@jbphet For a wrapper that provides sound levels and basic/enhanced options, all but the "current" sound will be in "basic" to start. "Enhanced" will include everything.

**Comment by jbphet** [comment]
I see this is assigned to me, but the description sounds like it should be assigned to Ashton.  I can add him to the GitHub project and assign him if needed.  It sounds like once the sound design is complete, we would create a new issue where I'd integrate the sounds and assign them to sonification levels.  Does that sound about right @emily-phet?

**Comment by emily-phet** [comment]
@jbphet I just wanted to give you a heads up that this was happening.

**Comment by jessegreenberg** [comment]
It looks like there is a more recent issue to review final sound designs, #120. @emily-phet @jbphet can this be closed?

**Comment by emily-phet** [comment]
Yup - this is old. Closing.
